src/scenes/Activities/ActivityHistory/EventRows/index.js

```diff
@@ -57,9 +59,8 @@ class EventRow extends Component {
         address = transaction.eventAddress;
         name = transaction.resultIndex;
       } else {
-        // TODO: fix this, these are both string values.
-        // need to convert to BN instances, add, then .toString()
-        amount = transaction.winningAmount + transaction.escrowWithdrawAmount;
+        amount = stringToBN(transaction.winningAmount) + stringToBN(transaction.escrowWithdrawAmount);
```

Comment:
after changing to `BN` instances, you need to use the `.add()` func. see BN docs.